hack/terraform-quickstart/start-cluster.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I don't have a strong desire to make this the case, but it would be nice. I looked at https://www.terraform.io/docs/provisioners/local-exec.html but I didn't see an easy way to trigger a script specifically until all the resources were up. It looks like in that example its tied to a specific resource.